.gitignore

```diff
@@ -1,5 +1,6 @@
 # maven ignore
 target/
+gen/
```

Comment:
What is useful of this folder?

Reply:
? You suggested to add it to gitignore, didn't you? BTW, it is for code generating from antlr.